### PR TITLE
New folder structure

### PR DESCRIPTION
**Pull request by aarajh**
- Modified files with the folder structure we agreed on.
- Added capability to have absolute imports

### CODE REVIEW

File: pitch/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "braces": ">=2.3.1",
     "expo": "^37.0.0",
     "formik": "^2.1.4",
+    "logkitty": "^0.7.1",
```
Comment (adil1508) [inline pitch/package.json:16]: is this some kinky shit youre into @aarajh ?

File: pitch/src/screens/Auth/NewPassword/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import NewPassword from './newpassword'
+// import enhance from './home.enhancer'
```
Comment (adil1508) [inline pitch/src/screens/Auth/NewPassword/index.js:2]: do we want this commented out?

Comment (aarajh) [reply pitch/src/screens/Auth/NewPassword/index.js:2]: for now its ok i think, when we actually have something to connect to redux, we will add it?

File: pitch/package.json (hunk 1 above)
Comment (aarajh) [reply pitch/package.json:16]: Security update :P

**Pull request merged**